### PR TITLE
Update debug-service.md --- add ipvs part on kube-proxy

### PR DESCRIPTION
**Pull request by islinwb**

### CODE REVIEW

File: content/en/docs/tasks/debug-application-cluster/debug-service.md [hunk 1]
```diff
@@ -500,6 +500,22 @@ There should be 1 rule in `KUBE-SERVICES`, 1 or 2 rules per endpoint in
 per endpoint, and a few rules in each `KUBE-SEP-(hash)` chain.  The exact rules
 will vary based on your exact config (including node-ports and load-balancers).
 
+#### Ipvs
```
Comment (m1093782566) [inline content/en/docs/tasks/debug-application-cluster/debug-service.md]: nit: Ipvs -> IPVS.

File: content/en/docs/tasks/debug-application-cluster/debug-service.md [hunk 2]
```diff
@@ -500,6 +500,22 @@ There should be 1 rule in `KUBE-SERVICES`, 1 or 2 rules per endpoint in
...
+  -> RemoteAddress:Port           Forward Weight ActiveConn InActConn
+...
+TCP  10.0.0.92:80 rr
+  -> 172.17.0.2:937               Masq    1      0          0
+  -> 172.17.0.4:937               Masq    1      0          0
+  -> 172.17.0.5:937               Masq    1      0          0
+...
+```
+
+There should be rules with "`clusterIP:Port` -> `podIP:Port`".
```
Comment (m1093782566) [inline content/en/docs/tasks/debug-application-cluster/debug-service.md]: Please explain it a bit more.

IPVS proxy will create a virtual server for each service address(e.g. Cluster IP, external IP, NodePort IP, Load Balancer IP etc.) and some corresponding real servers for endpoints of the service, if any.

**Comment by m1093782566** [comment]
LGTM from technical review.

**Comment by m1093782566** [comment]
@rramkumar1 You want to review this PR.

**Comment by rramkumar1** [comment]
/lgtm
This is a good start so let's just get this in. I think both the IPVS and the iptables sections could be expanded though.

**Pull request merged**